robonomics_liability/launch/liability.launch

```diff
@@ -5,6 +5,7 @@
   <arg name="factory_contract" default="factory.0.robonomics.eth" />
   <arg name="lighthouse_contract" default="airalab.lighthouse.0.robonomics.eth" />
   <arg name="enable_executor" default="true" />
+  <arg name="recording_topics" default="[]" />
```

Comment:
It's wildcard default behavior or nothing?

Reply:
By default will be recorded all topics like before.